src/main/java/com/google/api/codegen/transformer/BatchingTransformer.java

```diff
@@ -131,6 +147,41 @@ private BatchingDescriptorClassView generateDescriptorClass(GapicMethodContext c
     return desc.build();
   }
 
+  private BatchingDescriptorClassView generateDescriptorClass(DiscoGapicMethodContext context) {
```

Comment:
This looks the same as the method above - isn't the purpose of making the general `MethodContext` to allow for methods like this to be used by both Gapic and DiscoGapic?